lib/src/player_event_handler.dart

```diff
@@ -141,10 +141,11 @@ mixin PlayerEventHandler implements PlayerListener {
       case 'onCastTimeUpdated':
         emit(CastTimeUpdatedEvent.fromJson(data));
         break;
-      case 'onAirPlayAvailableEvent':
+      case 'onAirPlayAvailable':
+      case 'onAirplayAvailable':
```

Comment:
We found out there is a typo on iOS SDK and decided to keep both cases here to prevent future compatibility issues